pkg/modelzooserver/modelzooserver.go

```diff
@@ -15,28 +15,162 @@ package modelzooserver
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"log"
 
+	"sqlflow.org/sqlflow/pkg/database"
 	pb "sqlflow.org/sqlflow/pkg/proto"
 )
 
+const modelCollTable = "sqlflow_model_zoo.model_collection"
+const modelDefTable = "sqlflow_model_zoo.model_definition"
+const trainedModelTable = "sqlflow_model_zoo.trained_model"
+
+// TODO(typhoonzero): create tables if these tables are not pre created?
+const createTableStmts = `CREATE DATABASE IF NOT EXISTS sqlflow_model_zoo;
+DROP TABLE IF EXISTS sqlflow_model_zoo.trained_model;
+DROP TABLE IF EXISTS sqlflow_model_zoo.model_definition;
+DROP TABLE IF EXISTS sqlflow_model_zoo.model_collection;
+
+CREATE TABLE sqlflow_model_zoo.model_collection (
+    id INT AUTO_INCREMENT,
+    name VARCHAR(255),
+    version VARCHAR(255),
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE sqlflow_model_zoo.model_definition (
+    id INT AUTO_INCREMENT,
+    model_coll_id INT,
+    class_name VARCHAR(255),
+    args_desc VARCHAR(255),
+    PRIMARY KEY (id),
+    FOREIGN KEY (model_coll_id) REFERENCES model_collection(id)
+);
+
+CREATE TABLE sqlflow_model_zoo.trained_model (
+    id INT AUTO_INCREMENT,
+    model_def_id INT,
+    name VARCHAR(255),
+    version VARCHAR(255),
+    url VARCHAR(255),
+    description TEXT,
+    metrics TEXT,
```

Comment:
TEXT's max length is 64K, I'm not sure if it is enough for us.

Reply:
Should be enough, metrics field stores a JSON string of the model validation metrics like: 
```json
{"accuracy": 0.9, "AUC": 0.78}
```